ember-cli-build.js

```diff
@@ -65,7 +74,17 @@ module.exports = function() {
       'htmlbars-syntax':       htmlbarsPackage('htmlbars-syntax'),
       'simple-html-tokenizer': htmlbarsPackage('simple-html-tokenizer'),
       'htmlbars-test-helpers': htmlbarsPackage('htmlbars-test-helpers', { singleFile: true }),
-      'htmlbars-util':         htmlbarsPackage('htmlbars-util')
+      'htmlbars-util':         htmlbarsPackage('htmlbars-util'),
+      'glimmer-compiler':      glimmerPackage('glimmer-compiler'),
+      'glimmer-object':        glimmerPackage('glimmer-object'),
+      'glimmer-reference':     glimmerPackage('glimmer-reference'),
+      'glimmer-runtime':       glimmerPackage('glimmer-runtime'),
+      'glimmer-syntax':        glimmerPackage('glimmer-syntax'),
+      'glimmer-test-helpers':  glimmerPackage('glimmer-test-helpers'),
+      'glimmer-util':          glimmerPackage('glimmer-util'),
+      'glimmer-wire-format':   glimmerPackage('glimmer-wire-format'),
+      'glimmer':               glimmerPackage('glimmer'),
+      'simple-html-tokenizer': glimmerPackage('simple-html-tokenizer')
```

Comment:
Looks like we did some import changes in SHT: https://github.com/tildeio/simple-html-tokenizer/commits/master (htmlbars uses c4787e3 and glimmer uses 8b09fc2). If we can change htmlbars to work with the new structure then we can just release a new SHT version and use that in htmlbars (and do another htmlbars release?).